src/main/asciidoc/repository-query-keywords-reference.adoc

```diff
@@ -2,11 +2,32 @@
 [appendix]
 = Repository query keywords
 
-== Supported query keywords
-The following table lists the keywords generally supported by the Spring Data repository query derivation mechanism. However, consult the store-specific documentation for the exact list of supported keywords, because some keywords listed here might not be supported in a particular store.
+[[appendix.query.method.subject]]
+== Supported query method subject keywords
 
-.Query keywords
-[options="header", cols="1,3"]
+The following table lists the subject keywords generally supported by the Spring Data repository query derivation mechanism to express the predicate.
+Consult the store-specific documentation for the exact list of supported keywords, because some keywords listed here might not be supported in a particular store.
+
+.Query subject keywords
+[options="header",cols="1,3"]
+|===============
+|Keyword | Description
+|`find…By`, `read…By`, `get…By`, `query…By`, `search…By`, `stream…By`| General query method returning typically the repository type, a `Collection` or `Streamable` subtype or a result wrapper such as `Page`, `GeoResults` or any other store-specific result wrapper.
```

Comment:
Because of the ellipses, these items could benefit from an example or two. I wouldn't add the examples to the table but put them in a paragraph after the table. They're probably obvious enough for most users, but we can help out the new users a bit with a couple of examples.

Reply:
Thanks. That's fixed now in the final merge.